fli-iam/shanoir-ng
# #1856 - add mr dataset nature values

Closes #1856 

How to test:
- Update a study card and check new values are present
- Update this study card with new value of MrDatasetNature
- Import data and check value on dataset
